cmd/karma/views.go

```diff
@@ -424,6 +424,9 @@ func alerts(w http.ResponseWriter, r *http.Request) {
 					for _, am := range alert.Alertmanager {
 						for _, silence := range am.Silences {
 							if _, found := silences[am.Cluster][silence.ID]; !found {
+								if _, found :=  silences[am.Cluster]; !found {
```

Comment:
This should not be needed since all clusters are populated on https://github.com/prymitive/karma/blob/5e90c9df5bdbabcc41d51ab85b2d0303722c02f3/cmd/karma/views.go#L251-L257

Can you share your config?

Reply:
Here is our config. I changed IPs and names. And we have 10 alertmanagers:

```
alertmanager:
  interval: 30s
  servers:
    - name: my-test-monitoring01
      uri: http://10.0.0.1:9093
      timeout: 60s
      proxy: true
      readonly: false
    - name: my-test1-monitoring01
      uri: http://10.0.0.2:9093
      timeout: 60s
      proxy: true
      readonly: false
    - name: my-test2-monitoring01
      uri: http://10.0.0.4:9093
      timeout: 60s
      proxy: true
      readonly: false
    - name: my-test3-monitoring01
      uri: http://10.0.0.5:9093
      timeout: 60s
      proxy: true
      readonly: false
    - name: my-test4-monitoring02
      uri: http://10.0.0.6:9093
      timeout: 60s
      proxy: true
      readonly: false
annotations:
  default:
    hidden: false
  hidden: []
  visible: []
#debug: true
filters:
  default: []
karma:
  name: karma
listen:
  address: "127.0.0.1"
  port: 8080
  prefix: /karma/
log:
  config: false
  level: info
receivers:
  keep: []
  strip: []
ui:
  refresh: 60s
  hideFiltersWhenIdle: true
  colorTitlebar: false
  minimalGroupWidth: 420
  alertsPerGroup: 5
  collapseGroups: collapsedOnMobile
  theme: auto

```